torch_nets/smoke_test.py

```diff
@@ -0,0 +1,14 @@
+"""A smoke test for the ANN model."""
```

Comment:
Could this be a bit more descriptive about what this smoke test does and why it is useful please?

Reply:
I've extended the docstring

Reply:
Better, thank you.